README.md

```diff
@@ -34,9 +47,9 @@ This repo contains an end-to-end configuration for building a full X12 EDI syste
    npm run bootstrap
    ```
 
-## Testing the system
+# Testing the system
```

Comment:
Is this correct? This puts _Testing the system_ on the same level as _Stedi EDI Bootstrap_. I think it was correct before.